libs/design/Date/DateSummery.stories.tsx

```diff
@@ -6,7 +6,7 @@ export default {
   component: DateSummery,
   args: {
     endDate: undefined,
-    startDate: new Date,
+    startDate: new Date(),
```

Comment:
Let's not use `new Date()` as it gets updated on every render and creates fake changes.

Reply:
should we use `faker js`?

Reply:
No just hard code a value